drivers/SmartThings/zwave-thermostat/profiles/qubino-flush-thermostat-cooling.yml

```diff
@@ -6,6 +6,10 @@ components:
     version: 1
   - id: thermostatCoolingSetpoint
     version: 1
+    config:
+      values:
+        - key: "coolingSetpoint.value"
+          range: [ -10, 80 ]
```

Comment:
Shouldn't these be [-25, 80C] to match the requested [-13, 176F] range from the DTH?
https://smartthings.atlassian.net/browse/CHAD-8813?focusedCommentId=1224688

Reply:
should be fixed